skills/bear-clean-style/references/naming.md

```
@@ -0,0 +1,64 @@
+- Read: `<Entity>QueryInterface`
+- Write: `<Entity>CommandInterface`
+- Never mix read methods into Command interfaces or write methods into Query interfaces.
+
+## Read method names
+
+| Purpose | Method | SQL id / file |
+|---|---|---|
+| Primary-key item | `item(int $id)` | `<entity>_item` / `<entity>_item.sql` |
+| Natural key item | `bySlug`, `byEmail`, `byFilename` | `<entity>_by_slug.sql` etc. |
+| Collection | `list()` | `<entity>_list.sql` |
+| Filtered collection | `listBy<Article|Author|...>()` | `<entity>_list_by_<x>.sql` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)